src/Tribe/Cache.php

```diff
@@ -180,6 +181,36 @@ public function delete_expired_transients() {
 				AND a.option_name NOT LIKE '\_transient\_timeout_tribe\_%'
 		";
 		$wpdb->query( $sql );
+
+		if ( wp_using_ext_object_cache() ) {
+			/**
+			 * Toggle trigger to control if WordPress cache should be flushed or not.
+			 *
+			 * When using real object cache we cannot selectively flush a specific cache group.
+			 * This filter allows third-party code to handle the flush request in a more precise way by flushing only
+			 * the persistent cache group.
+			 *
+			 * @since TBD
+			 *
+			 *
+			 * @param bool   $flush Whether to flush the WordPress cache or not.
+			 * @param string $group The name of the group, `tribe-events`, we want to flush.
+			 */
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 0336cb59f27bbf3362fb33b89c204125</sub>